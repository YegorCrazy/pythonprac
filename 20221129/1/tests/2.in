class C(metaclass=dump):
	def __init__(self, a, b):
		self.a = a
		self.b = b
	def perf(self, op='+'):
		if op == '+':
			return self.a + self.b
		elif op == '-':
			return self.a - self.b
		else:
			 return None

c = C(2, 3)
print(c.perf())
print(c.perf(op='+'))
print(c.perf(op='-'))
print(c.perf(op='*'))
