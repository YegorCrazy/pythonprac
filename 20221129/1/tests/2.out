__init__: (2, 3), {}
perf: (), {}
5
perf: (), {'op': '+'}
5
perf: (), {'op': '-'}
-1
perf: (), {'op': '*'}
None
